Description: Enable display manager module, allowing autologins to work 
   * Enable displaymanager module, fixing autologin options
     (Closes: #934503, #934504)
Author: Jonathan Carter <jcc@debian.org>
Bug-Debian: https://bugs.debian.org/934503
Bug-Debian: https://bugs.debian.org/934504
Last-Update: 2020-07-15

--- /dev/null
+++ calamares-settings-debian-10.0.20/calamares/modules/displaymanager.conf
@@ -0,0 +1,28 @@
+# Configure one or more display managers (e.g. SDDM)
+# with a "best effort" approach.
+---
+#The DM module attempts to set up all the DMs found in this list, in that precise order.
+#It also sets up autologin, if the feature is enabled in globalstorage.
+#The displaymanagers list can also be set in globalstorage, and in that case it overrides anything set up here.
+displaymanagers:
+  - slim
+  - sddm
+  - lightdm
+  - gdm
+  - mdm
+  - lxdm
+  - kdm
+
+#Enable the following settings to force a desktop environment in your displaymanager configuration file:
+#defaultDesktopEnvironment:
+#    executable: "startkde"
+#    desktopFile: "plasma"
+
+#If true, try to ensure that the user, group, /var directory etc. for the
+#display manager are set up correctly. This is normally done by the distribution
+#packages, and best left to them. Therefore, it is disabled by default.
+basicSetup: false
+
+#If true, setup autologin for openSUSE. This only makes sense on openSUSE
+#derivatives or other systems where /etc/sysconfig/displaymanager exists.
+sysconfigSetup: false
--- calamares-settings-debian-10.0.20.orig/calamares/settings.conf
+++ calamares-settings-debian-10.0.20/calamares/settings.conf
@@ -36,6 +36,7 @@ sequence:
   - keyboard
   - localecfg
   - users
+  - displaymanager
   - networkcfg
   - hwclock
   - services-systemd
